app/components/Swapbox.tsx

```diff
@@ -230,6 +238,28 @@ export const Swapbox = ({ fixedProductMarketMaker }: SwapboxProps) => {
     !!connectorChainId &&
     !supportedChains.some(supportedChain => supportedChain.id === connectorChainId);
 
+  const priceInUSD = collateralTokenUSDPrice
+    ? swapDirection === SwapDirection.BUY
+      ? formatValueWithFixedDecimals(collateralTokenUSDPrice, 2)
+      : formatValueWithFixedDecimals(
+          collateralTokenUSDPrice * +currentState.tokenPrice,
+          2
+        )
+    : null;
+
+  const potentialProfit =
+    tokenAmountOut && tokenAmountIn && swapDirection === SwapDirection.BUY
```

Comment:
`const swapDirection = SwapDirection.BUY`

```suggestion
    tokenAmountOut && tokenAmountIn && isBuying
```